lib/budo.js

```diff
@@ -57,12 +57,8 @@ module.exports = function ({
   const budoFiles = []
 
   files.map((file) => {
-    if (file[1].split('.').pop() === 'css') {
-      transformCss({
-        filename: file[0],
-        outfile: file[1],
-        watch: true
-      })
+    if (file[0].indexOf('.css') !== -1) {
```

Comment:
Checking the file extension by using `String.indexOf` will not work for certain filenames, ex: `somefile.jsstyle.css` or `somefile.cssstuff.js`.